tests/test_sari.py

```diff
@@ -7,80 +7,81 @@
 
 
 @pytest.mark.skip(reason='TODO: Test probably broken, need to investigate')
-def test_sari_sentence():
+def test_sentence_sari():
     orig_sent = "About 95 species are currently accepted ."
     refs_sents = ["About 95 species are currently known .",
-                 "About 95 species are now accepted .",
-                 "95 species are now accepted ."]
+                  "About 95 species are now accepted .",
+                  "95 species are now accepted ."]
 
     sys_sent = "About 95 you now get in ."
-    sari_score = sari.sari_sentence(orig_sent, sys_sent, refs_sents)
+    sari_score = sari.sentence_sari(orig_sent, sys_sent, refs_sents, tokenizer='none')
     assert sari_score == pytest.approx(26.82782411698074)
 
     sys_sent = "About 95 species are now agreed ."
-    sari_score = sari.sari_sentence(orig_sent, sys_sent, refs_sents)
+    sari_score = sari.sentence_sari(orig_sent, sys_sent, refs_sents, tokenizer='none')
     assert sari_score == pytest.approx(58.89995423074248)
 
     sys_sent = "About 95 species are currently agreed ."
-    sari_score = sari.sari_sentence(orig_sent, sys_sent, refs_sents)
+    sari_score = sari.sentence_sari(orig_sent, sys_sent, refs_sents, tokenizer='none')
     assert sari_score == pytest.approx(50.71608864657479)
 
 
-def test_corpus_sari_plain():
+@pytest.mark.skip(reason='TODO: Test probably broken, need to investigate')
```

Comment:
Do you have an idea of why it's broken?

Reply:
It's not really 'broken'. If you see the code, it's exactly the same as for test_corpus_sari_legacy. We need to come up with a new test for when we use the detokenised and recased versions of both the test sets and the system outputs. I'm going to delete the test.